### DISCUSSION TITLE
[4.2][Feature request] Dark Mode

### DISCUSSION THREAD

**Discussion by tabacitu** [Laravel-Backpack/community-forum#335] [Ideas]
Since MacOS & iOS have dark mode too now, it could make sense to create a new overlay for dark mode, and have a use toggle between Auto, Light and Dark.

https://nandovieira.com/supporting-dark-mode-in-web-content

Of course, dark mode websites look pretty bad right now - design-wise they’re still a long way from light designs.

But we _could_ do this... Food for thought. And feedback.

**Comment by martijnb92** [comment]
Great idea!
It would be really nice to have a dark mode in Backpack.
Some things to think about before working on a dark mode overlay:

- Will Backstrap be upgraded to CoreUI 3?
- Will this cause issues with CoreUI since they have a dark layout included in the Pro version?
- How to deal with the different color schemes currently available?
